### DISCUSSION TITLE
libgccjit

### DISCUSSION THREAD

**Discussion by podczeck** [Homebrew/discussions#1448] [Everyday usage]
I installed libgccjit from homebrew on the latest MacOS.

Using the option --with-native-compilation to build Emacs, during the configure phase I got:

checking libgccjit.h usability... yes
checking libgccjit.h presence... yes
checking for libgccjit.h... yes
configure: error: Installed libgccjit has failed passing the smoke test.
...
Please report the issue to your distribution if libgccjit was installed through
that.

**Comment by carlocab** [comment]
What compiler are you using? Do you have Homebrew `gcc` installed?

**Reply by podczeck** [reply]
I use the compiler from Xcode.app. Is this the mistake?

**Reply by carlocab** [reply]
Do you have Homebrew `gcc` installed?

**Reply by podczeck** [reply]
I had not. After I installed gcc from Homebrew, the problem with the smoke test has gone. Thanks for your quick help!
